mizar/common/rpc.py

```diff
@@ -149,6 +151,27 @@ def update_ep(self, ep):
             self.update_port(ep.get_tunnel_id(), ep.get_ip(),
                              frontend_ports[i], remote_ports[i], protocols[i])
 
+    def update_packet_metadata(self, ep):
+        peer = ""
+        droplet_ip = ep.get_droplet_ip()
+        # Only detail veth info if the droplet is also a host
+        if (droplet_ip and self.ip == droplet_ip):
+            peer = ep.get_veth_peer()
+
+        jsonconf = {
+            "tunnel_id": ep.get_tunnel_id(),
+            "ip": ep.get_ip(),
+            "hosted_iface": peer,
```

Comment:
What is this field used for?

Reply:
I don't think we need this peer field, it's already available in the endpoint table

Reply:
Deleted.